wgsl/index.bs

```diff
@@ -329,6 +339,16 @@ A vector type is a <dfn dfn>numeric vector</dfn> type if its component type is a
         Equivalently, it can be viewed as |N| column vectors of type vec|M|&lt;f32&gt;.
 </table>
 
+The key use case for a matrix is to embody a linear transformation.
+In this interpretation, the vectors of a matrix are treated as column vectors.
+
+The product operator (`*`) is used to either:
+
+* scale the transformation by a scalar magnitude.
+* apply the transformation to another matrix or vector.
```

Comment:
I think it could be cleaner to separate these cases into:
```
* apply the transformation to a vector
* combine the transformation with another matrix
```